README.md

```diff
@@ -38,6 +43,14 @@ The list of items you’ll need is also included below:
 
 ![](https://www.balena.io/blog/content/images/2018/11/image17.jpg)
 
+## Using WiFi Connect
+
+The balenaDash project includes [wifi-connect](https://github.com/balena-io/wifi-connect) which enables your device to operate as a WiFi access point and allow you to join a different WiFi network using captive portal functionality. Although you can specify a WiFi network to join when you first add your device and download the image from the balenaCloud dashboard, there may be situations where you need to change that.
+
+WiFi Connect periodically tests for a functional internet connection, if nothing is found the device sets itself up as a WiFi access point named `balenaDash` that you can join with a mobile device. **Note: due to the fact the built-in WebUI for balenaDash uses port 80, the default captive portal popup screen will not work.**
+
+To use WiFi Connect you need to join the network with a mobile device and ignore the captive portal popup. Ensure that you remain connected to the `balenaDash` network and visit the IP address of the device in a browser on port `8080`. For example `http://<ip of balenaDash device>:8080`. This will allow you to access WiFi Connect, perform a site survey and join a different WiFi network.
```

Comment:
@chrisys is there no way to have wifi connect run on port 80? I know @majorz had some ideas on how to do that.

Reply:
@shaunmulligan it would be great as the current process isn't exactly user-friendly. Is there any way to set preference you know of? I.e. if two containers expose the same port but there's an ordering, and therefore if wifi-connect is running it gets port 80 but otherwise falls back to the webui.